src/components/molecules/NavSearchBar/NavSearchResult.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+
+import { UserAvatar } from "components/atoms/UserAvatar";
+
+import "./NavSearchResult.scss";
+
+export interface NavSearchResultProps {
+  title: string;
+  description?: string;
+  image?: string;
+  isAvatar?: boolean;
+  onClick?: (e?: React.MouseEvent) => void;
+}
+
+export const NavSearchResult: React.FC<NavSearchResultProps> = ({
+  title,
+  description,
+  image,
+  isAvatar,
+  onClick,
+}) => {
+  const imageStyles = {
+    backgroundImage: image ? `url(${image})` : undefined,
+  };
+
+  return (
+    <div className="NavSearchResult font-size--small" onClick={onClick}>
+      {isAvatar ? (
+        <UserAvatar
+          avatarSrc={image}
+          containerClassName="NavSearchResult__avatar"
+        />
+      ) : (
+        <div className="NavSearchResult__image" style={imageStyles} />
+      )}
+
+      <div>
+        <div>{title}</div>
+        <div className="font-size--small NavSearchResult__description">
+          {description}
+        </div>
+      </div>
+    </div>
```

Comment:
I feel like in cases where we have explicit classes assigned already (eg. `NavSearchResult`, `NavSearchResult__description`) it's better to have 'utility classes' inlined into their definition, rather than as additional classes in the JSX (eg. `font-size--small`) so that when we are looking at the styles for the class, we can see everything relevant to it.

We could probably have a 'best of both worlds' if we make SCSS mixins/etc for this that we can use in the explicitly named classes, and then could have these util classes for "things that otherwise don't have their own direct styling/classes". The util classes would be defined using the mixins as well.

Reply:
See also: https://github.com/sparkletown/internal-sparkle-issues/issues/4#issuecomment-824454619 / https://github.com/sparkletown/sparkle/pull/1323#discussion_r617982742

Reply:
do you think it'll make sense to use `$font-size--small` variable instead? or should we just simply inline the value?

Reply:
That's where I was thinking that mixins/functions/similar would potentially be useful. At the very least I think a variable for it.

> We could probably have a 'best of both worlds' if we make SCSS mixins/etc for this that we can use in the explicitly named classes, and then could have these util classes for "things that otherwise don't have their own direct styling/classes". The util classes would be defined using the mixins as well.

Reply:
using the variable: https://github.com/sparkletown/sparkle/pull/1328/commits/5afa888773470274a321c8b6e1ba1e547b72776c